index.ts

```diff
@@ -390,10 +423,22 @@ export class OAuth2AuthCodePKCE {
       }
 
       localStorage.setItem(LOCALSTORAGE_STATE, JSON.stringify(this.state));
-      return { token: accessToken, scopes };
+
+      if (Object.keys(tokensToExpose).length > 0) {
+        return {
+          explicitlyExposedTokens: tokensToExpose,
+          token: accessToken,
+          scopes
+        };
+      }
+
+      return {
+        token: accessToken,
+        scopes
+      };
```

Comment:
There's no need to repeat the same structure twice: could you please change this so it just adds the property to the existing object?

Such as:
```
let accessContext = { token, scopes };
if (Object.keys(tokensToExpose).length > 0) { accessContext.explicitlyExposedTokens = tokensToExpose; }
return accessContext;
```
It reduces a lot of noise :slightly_smiling_face: